src/components/AcceptTask/FileAttachmetCard.tsx

```diff
@@ -4,17 +4,26 @@ import styles from './index.module.scss';
 
 interface IFileAttachmentCard {
   label: string;
+  skills?: string;
 }
 
-const FileAttachmentCard: FC<IFileAttachmentCard> = ({ label }) => {
+const FileAttachmentCard: FC<IFileAttachmentCard> = ({ label, skills }) => {
```

Comment:
What's skills doing in this component?

Reply:
I didn't want to use separate component for skills. This way, we can show skills and attachments using the same component.